common/src/test/java/com/tc/runtime/MemoryPoolsTest.java

```diff
@@ -33,6 +33,7 @@ public MemoryPoolsTest() {
   }
 
   public void testMemoryPools() throws Exception {
+    // This test fails with java11 because of collection count assertion failure .
```

Comment:
This test is dependent on the GC implementation being used.  Under Java 8, ParallelGC is used -- this implementation responds positively to a `System.gc()` call so the test works.  Under Java 11, G1GC is the default -- this implementation checks to see if it thinks a GC is necessary and, under conditions of the test, doesn't perform a GC -- so the test fails.  The test either needs to apply some heap load so a GC is necessary or needs to check something else when G1GC is being used and test something else (or skip the test entirely).